### DISCUSSION TITLE
Structure lost on export?

### DISCUSSION THREAD

**Discussion by ConwayMatt** [vzhd1701/evernote-backup#93] [General]
This is a pretty great looking tool - ran flawlessly for me on the first go.  Thank you! 
The one thing I was hoping for was to maintain the structure I had in Evernote,  but this tool seems to flatten the very little hierarchy I have. 

What I see is this:  every Stack in EN (a collection of notebooks) turns into a folder and then every notebook in that stack is traversed, exporting every note, before moving on to the next notebook in that stack, but the thing is, the notebook itself isn't preserved; everything is dumped into the folder that corresponds to an EN stack. 

What I would expect:   Every Stack turns into a folder.   Every notebook in that stack would turn into a subfolder underneath the stack-folder.   Every note in that notebook would turn into a note in the subfolder.   Stack->Notebook->Note as the output, not Stack->All notes. 

Perhaps I'm doing something wrong or is this the behavior of the tool.   
If the latter, how hard would it be to allow for the preservation of the notebooks-as-folders on export? 

thanks!
M

**Comment by vzhd1701** [comment]
To get this type of structure you need to use `--single-notes` option during export.
